Fhi.HelseId/Fhi.HelseId.csproj

```diff
@@ -6,13 +6,13 @@
     <LangVersion>8</LangVersion>
     <Configurations>Debug;Release</Configurations>
     <id>Fhi.HelseId</id>
-    <Version>2.3.1</Version>
+    <Version>2.4.0</Version>
     <authors>Folkehelseinstituttet (FHI), Nasjonalt Helsenett (NHN)</authors>
     <projectUrl>https://github.com/folkehelseinstituttet/Fhi.HelseId</projectUrl>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <icon>https://github.com/folkehelseinstituttet/Fhi.HelseId/images/fhi.png</icon>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
-    <PackageReleaseNotes>Initial release</PackageReleaseNotes>
+    <PackageReleaseNotes>Split altinn out</PackageReleaseNotes>
```

Comment:
kanskje være litt mer eksplisitt:
"Move Altinn functionality to separate package Fhi.HelseId.Altinn"

Reply:
Done